### ISSUE TITLE
decimal point vs decimal comma

### ISSUE THREAD

**Issue by ANELKAOS** [galculator/galculator#50]
Hello,

The decimal point and the decimal comma are both ISO valid decimal separators. See https://www.iso.org/standard/31887.html for details.

When adding two floating-point numbers in Calculator > Notation mode > Formula Entry you round to the nearest lowest value in case you use a comma:  
WRONG (galculator): 123,11+123,89 = 246
RIGHT: 123.11+123.89 = 247 (accurate, note the use of periods)

Following the IEEE 754-2008, galculator is rounding between floating points separated by a comma, and separated by a period in a different way. It works on Algebraic notation mode. Unfortunately, it could cause inaccuracy on formulas, e.g., calculus of your favorite stock:

Stock price: 1,99
Fee per stock: 0,99
Number of stocks: 1000000
Formula: (1,99+0,99)*1000000 = 1000000 You got 2M of discount! So wrong.

Please, fix it.

Best regards.

**Comment by Mi605** [comment]
Yes, me also strongly encourages you to fix this issue. Since you have translated the project to many languages, you should make sure to make use of the proper locale settings for decimal separator and comma also.  You can retrieve them easily from system environment (locale settings). Please be aware the US style (using point for comma) is not common in the rest of the world and causes wrong results when using galculator. So what keeps you from respecting the users' locale setting, more than a year later now?
